docs/make.jl

```diff
@@ -154,12 +154,8 @@ Documenter.DocMeta.setdocmeta!(
     linkcheck_ignore = [
         # Ignore the PDF link, because it hasn't been built yet.
         "MathOptInterface.pdf",
-        # Ignore tags, because prepping for a new release will otherwise cause
-        # it to fail.
-        r"https://github.com/jump-dev/MathOptInterface.jl/releases/tag/v([0-9]).([0-9]+).([0-9]+)",
-        # Ignore issue and pull request links, because there are many of them,
-        # and they sometimes time-out the linkcheck.
-        r"https://github.com/jump-dev/MathOptInterface.jl/issues/([0-9]+)",
+        # Ignore the very many GitHub links
+        r"https://github.com/jump-dev/.+",
```

Comment:
Because unauthenticated requests are now heavily rate limited: https://github.blog/changelog/2025-05-08-updated-rate-limits-for-unauthenticated-requests/

Reply:
cc @mortenpi is this known?

Reply:
I have seen `429`s in Documenter's own linkchecks, but I wasn't aware of the rate limit changes.

Reply:
We could special case GitHub and let it (optionally) pick up the `GITHUB_TOKEN` for it. I assume you might be able to work around the rate limit that way.

Reply:
Authenticating with our `GITHUB_TOKEN` fixed some 429 in JuMP: https://github.com/jump-dev/JuMP.jl/pull/4008

I'll open an issue in Documenter

Reply:
https://github.com/JuliaDocs/Documenter.jl/issues/2727